buildSrc/src/main/kotlin/com/datadog/gradle/config/NightlyTestsCoveragConfig.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.gradle.config
+
+import org.gradle.api.Project
+import org.gradle.api.tasks.Exec
+
+const val TEST_COVERAGE_THRESHOLD = 0.54
```

Comment:
This value should not be harcoded in the `buildSrc` module. Instead it should be in a `build.gradle/kts` file.
I would also define this task in a gradle script file too as it's only used in one place, but you can keep it here if it's more convenient

Reply:
I'd rather keep it here for now. I made the threshold a variable of that function.